### ISSUE TITLE
CodeView: Box title

### ISSUE THREAD

**Issue by a12jiyba** [HGustavs/LenaSYS#992]
the first letter in the box title is not showing correctly when we use more than 2 boxes.

**Comment by HGustavs** [comment]
I believe this could be related to the resize code that is called automatically. It would be nice to know if the templates work without this code and if so, remove it.

```
$(window).resize(function() {
	var windowHeight = $(window).height();
	textHeight= windowHeight-50;
	$("#table-scroll").css("height", textHeight);

	// Keep right margin to boxes when user switch from mobile version to desktop version
	if($(".buttomenu2").height() == null){
		var boxmenuheight = 0;
	}else{
		var boxmenuheight= $(".buttomenu2").height();
	}
	$(".box").css("margin-top", boxmenuheight);

});
```
I feel wary about css templates that require javascript/jquery shims to function properly.

**Comment by f12danjo** [comment]
This was fixed by pull request [#1176](https://github.com/HGustavs/LenaSYS/pull/1176)
